### ISSUE TITLE
Enlargement without wrapping is not accessibility support

### ISSUE THREAD

**Issue by WayneEDick** [w3c/wcag21#277]
WCAG 2.0 based its enlargement concepts on a fallacy. Namely,
Enlargement without word wrapping is a  accessible support. This is false for two reasons:
1. The amount of operations required to perceive text is excessive in the extreme.
2. The character buffer that the user can have present for cognitive recall is too small.

Now that we know enlargement without word wrapping is not accessibility support, we must establish methods to enable sufficient enlargement that require word wrapping for all text.

In most cases this requires a full screen view. That means that a linear arrangement of content into one that can be viewed full-screen is essential to low vision access.

Consider the example of an integrated development environment. To a user with full sight the multi-column / multi-row  presentation is extremely useful. Many usability studies have shown that having tools present and visible make increases the user's report of satisfaction. The goal of an IDE is to facilitate many aspects of code development and having instant access to most of the key operations is a serious time saver.

Now consider the case of low vision. If the edit area is surrounded by task panels, panes and toolbars, there is not much room for large print text. This will require zoom technology and the cost of scrolling will overwhelm any time savings gotten from instant access to toolbars, panes and panels.  

A linearized view for edit and toolbars, panes and panels will be slower than the access for a person with full sight, but it will be tremendously faster than access using zoom with no wrap. Also it enables full screen viewing of toolbars, panes and panels. That would enable enough enlargement to see the content.

This probably seems unthinkable to a person with normal sight. You may say to yourself, "This cannot be." The fact is that the normal layout of IDE technology shuts people with low vision out of programming employment. 

Using the development environments used in Science, Technology, Engineering and Mathematics it is no surprise that one in 30,000 people with low vision obtain a degree in STEM. 

You can hammer as hard as you like. A square peg doesn't fit in a round hole.  If we did not know how to linearize, I wouldn't complain. But we know how. Our cellphone experience verifies that.

**Comment by patrickhlauke** [comment]
Small request: could you make the issue titles a bit more specific/descriptive, so it's clear what they actually refer to at first glance without having to read them?

**Comment by alastc** [comment]
As Wayne has retired I took the liberty of updating the title.

I think the new zoom criteria goes some way to address the meat of this, but the next action would be to look through WCAG 2.0 (mostly non-normative docs related to 1.4.4) and see what could be updated to address this concept.

**Comment by patrickhlauke** [comment]
as an aside: @alastc speaking of the new zoom SC, I trust you saw this? https://github.com/w3c/wcag21/issues/335

**Comment by alastc** [comment]
Yep, that went up whilst I was on holiday, just getting back to things, thanks.

**Comment by awkawk** [comment]
@WayneEDick the Zoom SC does require wrapping. If you have additional comments/concerns with the current version please let us know. Thanks for the comment!
